homeassistant/components/technove/switch.py

```diff
@@ -34,8 +49,22 @@ class TechnoVESwitchDescription(SwitchEntityDescription):
         translation_key="auto_charge",
         entity_category=EntityCategory.CONFIG,
         is_on_fn=lambda station: station.info.auto_charge,
-        turn_on_fn=lambda technoVE: technoVE.set_auto_charge(enabled=True),
-        turn_off_fn=lambda technoVE: technoVE.set_auto_charge(enabled=False),
+        turn_on_fn=lambda coordinator: coordinator.technove.set_auto_charge(
+            enabled=True
+        ),
+        turn_off_fn=lambda coordinator: coordinator.technove.set_auto_charge(
+            enabled=False
+        ),
+    ),
+    TechnoVESwitchDescription(
+        key="session_active",
+        translation_key="session_active",
+        entity_category=EntityCategory.CONFIG,
+        is_on_fn=lambda station: station.info.is_session_active,
+        turn_on_fn=lambda coordinator: _set_charging_enabled(coordinator, enabled=True),
+        turn_off_fn=lambda coordinator: _set_charging_enabled(
```

Comment:
Lambdas shouldn't be multiline

Reply:
It's a single function call. Do you suggest I make the variable names shorter so it can fit on a single line? It gets formatted this way automatically.

Reply:
Im not exactly sure about the motivations here. But for me when I could t fit a lambda ok one line i needed to either short it or define a method separately

Reply:
I replaced the lambda with a function def, `_enable_charging` and `_disable_charging`